### PR TITLE
Cleanup: console logging, debug and testing output, lint.

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**

### CODE REVIEW

File: src/Containers/CadView.jsx [hunk 1]
```diff
@@ -151,12 +151,12 @@ export default function CadView({
...
       transparent: true,
       opacity: 0.5,
-      color: theme.palette.highlight.light,
+      color: theme.palette.highlight.main,
       depthTest: true,
     })
     const selectMat = new MeshLambertMaterial({
       transparent: true,
-      color: theme.palette.highlight.main,
+      color: theme.palette.secondary.main,
```
Comment (OlegMoshkovich) [inline src/Containers/CadView.jsx:159]: this breaks high - lighting
image

Comment (pablo-mayrgundter) [reply src/Containers/CadView.jsx:159]: So the highlighting that is active in production was broken but in a different way.  The console log shows the highlight color is undefined.. it was being passed in from our theme, but the theme property wasn't defined.  So, I picked what looked good to me last night :) but yeah, open to any suggestions for alternatives

Comment (OlegMoshkovich) [reply src/Containers/CadView.jsx:159]: image

**Comment by pablo-mayrgundter** [comment]
Ptal

**Comment by OlegMoshkovich** [comment]
There are conflicts.

**Review by OlegMoshkovich — APPROVED** [review]

**Comment by pablo-mayrgundter** [comment]
Fixes #448

**Pull request merged**